cookbook/profiler/data_collector.rst

```diff
@@ -203,7 +203,7 @@ must also define additional blocks:
             </div>
         {% endset %}
 
-        {{ include('@WebProfiler/Profiler/toolbar_item.html.twig') }}
+        {{ include('@WebProfiler/Profiler/toolbar_item.html.twig', {'link': profiler_url}) }}
```

Comment:
@xabbuh however, the symfony core uses `profile_url` instead `true`:
* [Bundle/WebProfilerBundle/Resources/views/Collector/twig.html.twig](https://github.com/symfony/symfony/blob/2.7/src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/twig.html.twig#L27)
* [Bundle/WebProfilerBundle/Resources/views/Collector/translation.html.twig](https://github.com/symfony/symfony/blob/2.7/src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/translation.html.twig#L37)
* [Bundle/WebProfilerBundle/Resources/views/Collector/time.html.twig]
(https://github.com/symfony/symfony/blob/2.7/src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/time.html.twig#L30)
* etc...